core/src/main/java/org/apache/spark/api/java/function/package.scala

```diff
@@ -22,4 +22,4 @@ package org.apache.spark.api.java
  * these interfaces to pass functions to various Java API methods for Spark. Please visit Spark's
  * Java programming guide for more details.
  */
-package object function 
+package object function
```

Comment:
@rxin . Yep. It's not worth of it.
Let's forget about this for now.
Please never mind my last comment.